Mathlib/LinearAlgebra/Matrix/Spectrum.lean

```diff
@@ -54,12 +53,7 @@ lemma mulVec_eigenvectorBasis (j : n) :
 
 /-- The spectrum of a Hermitian matrix `A` coincides with the spectrum of `toEuclideanLin A`. -/
 theorem spectrum_toEuclideanLin : spectrum 𝕜 (toEuclideanLin A) = spectrum 𝕜 A :=
-  AlgEquiv.spectrum_eq
-    (AlgEquiv.trans
-      ((toEuclideanCLM : Matrix n n 𝕜 ≃⋆ₐ[𝕜] EuclideanSpace 𝕜 n →L[𝕜] EuclideanSpace 𝕜 n) :
-          Matrix n n 𝕜 ≃ₐ[𝕜] EuclideanSpace 𝕜 n →L[𝕜] EuclideanSpace 𝕜 n)
-      (Module.End.toContinuousLinearMap (EuclideanSpace 𝕜 n)).symm)
-    _
+  AlgEquiv.spectrum_eq (Matrix.toLinAlgEquiv (PiLp.basisFun 2 𝕜 n)) _
```

Comment:
It feels like there's some slightly surprising defeq here between `Matrix.toLinAlgEquiv (PiLp.basisFun 2 𝕜 n) A` and `toEuclideanLin A`, but in the likely event it does break the blame will now lead to this comment :).

bors merge